src/ng1/plugins/customscroll/jquery.jscrollpane.js

```diff
@@ -184,13 +199,13 @@ LICENSE-END
 					maintainAtBottom = settings.stickToBottom && isCloseToBottom();
 					maintainAtRight  = settings.stickToRight  && isCloseToRight();
 
-					hasContainingSpaceChanged = elem.innerWidth() + originalPaddingTotalWidth != paneWidth || elem.outerHeight() != paneHeight;
+					hasContainingSpaceChanged = elem.innerWidth() + originalPaddingTotalWidth != paneWidth || elem[0].outerHeight != paneHeight;
```

Comment:
Unfortunately we won't be able to use this solution. `outerHeight` is not a valid property on an element, it is only present on the window object:

![image](https://user-images.githubusercontent.com/20795331/31717707-ebb55c6a-b404-11e7-8470-d248f338b4ec.png)

With this change here the `outerHeight` value will always be different than the paneHeight value as it will always be undefined.